### PR TITLE
docs: add cocotb as requirement

### PR DESCRIPTION
**Pull request by rgantonio**
Adding cocotb as requirement for verification purposes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 [![Documentation Status](https://readthedocs.org/projects/hwpe-doc/badge/?version=latest)](https://hwpe-doc.readthedocs.io/en/latest/?badge=latest)
 
+# HWPE Stream 
+
+## Installation Requirements
```
Comment (JosseVanDelm) [inline README.md]: Add the required python version please

**Review by JosseVanDelm — CHANGES_REQUESTED** [review]
Cool Ryan! Can you please add the required python version

File: README.md (hunk 1 above)
Comment (rgantonio) [reply README.md]: Added! Done!

File: README.md [hunk 2]
```diff
@@ -2,7 +2,12 @@
 
 # HWPE Stream 
 
-## Installation Requirements
+## Installation Requirements\
```
Comment (JosseVanDelm) [inline README.md:5]:
```suggestion
## Installation Requirements
```
Would remove the backslash though

**Review by JosseVanDelm — APPROVED** [review]

**Pull request merged**